desktop/components/main_layout/header/view.coffee

```diff
@@ -126,3 +132,10 @@ module.exports = class HeaderView extends Backbone.View
       new FlashMessage message: 'Thank you for personalizing your profile'
     else if document.referrer.match '/artsy-primer-personalize/'
       new FlashMessage message: 'Thank you. Please expect your personalized portfolio in the next 2 business days.'
+
+  highlightSearch: (e) ->
+    if $('.html-highlight-header-search-split-test #main-layout-search-bar-input').is(':focus')
+      $('#main-layout-search-bar-container').addClass('focused')
```

Comment:
It may be possible to add a class to the input that has `:focus` to style the search bar without javascript, but I don't mind javascript either.